runtime/moonriver/src/lib.rs

```diff
@@ -1392,6 +1392,25 @@ impl pallet_parameters::Config for Runtime {
 	type WeightInfo = moonriver_weights::pallet_parameters::WeightInfo<Runtime>;
 }
 
+parameter_types! {
+	pub MbmServiceWeight: Weight = Perbill::from_percent(80) * RuntimeBlockWeights::get().max_block;
```

Comment:
I think we should unify this with `MaximumSchedulerWeight`.

Reply:
Like [this](https://github.com/moonbeam-foundation/moonbeam/pull/3224/commits/eda07de35a56c51920195e2a5f3ef25e3f0951f8) or is there a way for both parameters to share the weight?

Reply:
The change looks good 👍